### DISCUSSION TITLE
Experience with Monarc 2.12.6

### DISCUSSION THREAD

**Discussion by dbarzin** [monarc-project/MonarcAppFO#492] [Q&A] [answered]
Here is some issues I am facing with Monarc VM 2.12.6

I have download the VM from https://vm.monarc.lu/MONARC_v2.12.6@ddca7f3/ and imported it in VirtualBox

1. Updating Monarc

- I have used the procedure at  https://www.monarc.lu/documentation/technical-guide/#updates
- The folder to execute the update command is not specified  ./scripts/update-all.sh -c It has to be run in /var/lib/monarc/fo
- When updating I got the error message :
    fatal: not a git repository (or any of the parent directories): .git
    A problem occurred while retrieving remote files from repository.

2. Importing ISO 27002:2022
I cannot import the referential ISO 27001:2022 as describe in this video : https://www.youtube.com/watch?v=N45p3fw_G2U&ab_channel=CasesLuxembourg
The menu item “referential” is not present :
 
![image](https://github.com/monarc-project/MonarcAppFO/assets/43208232/8939381f-be12-41d0-a822-db26f5c02046)

4. Import
Importing a risk analysis from another Monarc instance generate an error message
« Une erreur est survenue sur api/client-anr/1/instances/import : Syntax error (500) »
 
![image](https://github.com/monarc-project/MonarcAppFO/assets/43208232/de2b4748-b0f6-40f0-9e8d-18d3252cc5d5)

Thanks for you help !

**Comment by jerolomb** [comment]
Hi,

1. if you have download the last version, you don't need to update it. 
2. It seems that's a very old MONARC, is it the one you download ? (if it's the case we will check but you can try the one from Github https://github.com/monarc-project/MonarcAppFO/releases/tag/v2.12.6) Because "Mesures 27002" is not part from the app since long time. Can you check the version on the main page ? 
4. Can you provide more details, from which version to which version ? Maybe some log you can send etc.

**Comment by dbarzin** [comment]
The link to the OVA on this page is incorrect https://www.monarc.lu/news/2023/04/25/monarc-2126-released/
That's the source of my problems.
Thanks,
Didier

**Reply by ruslanbaidan** [reply, marked as answer]
Updated the the link to the corresponding VM, sorry for the inconvenience.

**Comment by jerolomb** [comment]
Thanks Didier, we will investigate
